verifier/testreport.py

```diff
@@ -921,6 +927,28 @@ def check_simple_text_diffs(self, test_list, category):
 
         return dict(results)
 
+    def check_list_differences(self, test_list, category):
+        #TODO Check if expected and results are differences in list content
+        results = defaultdict(list)
+        all_checks = ['different lengths', 'different_content', 'other list difference']
+        for check in all_checks:
+            results[check] = set()
+
+        for fail in test_list:
+            label = fail['label']
+            actual = fail['result']
+            expected = fail.get('expected', None)
+            if type(actual) != type(expected):
+                # This is a type mismatch
+                continue
+            if type(actual) == list:
+                if len(actual) != len(expected):
+                    results['different lengths'].add(label)
+                else:
+                    results['other list difference'].add(label)
```

Comment:
idea(optional): maybe here, or maybe in the failure/error test characterization in the frontend, give an indication of the index number of the first element that diverges across the 2 compared lists, and also produce those element values.

Reply:
This is a useful thought for looking at the differences for a particular test between expected and result. However, making a categorization for each will be too specific. This could be handled by a function for hovering over the result, similar to that now used for string differences.